tests/__init__.py

```diff
@@ -66,5 +66,12 @@ def setUp(self):
         loader.data_path = ''
         self.loader = loader
 
+        # We also need to patch the global default session.
+        # Otherwise it could be a cached real session came from previous
+        # "functional" or "integration" tests.
+        _patch_global_session = mock.patch('boto3.DEFAULT_SESSION')
```

Comment:
These don't need the leading `_` prefix because they're only visible within this method anyways.